atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/TransactionManagerTest.java

```diff
@@ -193,10 +194,13 @@ private TransactionManager setupTransactionManager() {
         when(timelock.getFreshTimestamp()).thenReturn(1L);
         when(timelock.lockImmutableTimestamp(any())).thenReturn(
                 LockImmutableTimestampResponse.of(2L, LockToken.of(UUID.randomUUID())));
-        when(timelock.startAtlasDbTransaction(any())).thenReturn(
-                StartAtlasDbTransactionResponse.of(
-                        LockImmutableTimestampResponse.of(2L, LockToken.of(UUID.randomUUID())), 1L));
+        when(timelock.startIdentifiedAtlasDbTransaction(any())).thenReturn(
+                StartIdentifiedAtlasDbTransactionResponse.of(
+                        LockImmutableTimestampResponse.of(2L, LockToken.of(UUID.randomUUID())),
+                        TimestampAndPartition.of(1L, 1)
+                ));
         TRM.registerTransactionManager(txnManagerWithMocks);
+
```

Comment:
nit: newline?

Reply:
Yep, will remove this. Must have accidentally added it